students/uss_tomasz/lesson_06_dictionaries_tuples_sets_args_kwargs/6_6/args_sum.py

```diff
@@ -0,0 +1,8 @@
+def sum_all(*args):
+    total = 0
+    for i in args:
+        total += i
+    return total
```

Comment:
`reduce` could be used https://thepythonguru.com/python-builtin-functions/reduce/